addons/sale_stock/sale_stock.py

```diff
@@ -291,6 +305,25 @@ def action_done(self):
             line.qty_delivered = line._get_delivered_qty()
         return result
 
+    @api.multi
+    def assign_picking(self):
```

Comment:
The original method  assigns an existing picking or a new one on the move. So I think that we can reuse it on the override. Do not modify your branch, I'm working on it.

Thank you